docs/UserGuide.md

```diff
@@ -22,6 +22,12 @@ optimized features for Command Line Interface.
 1. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app
    contains some sample data.<br>
    ![Ui](images/Ui.png)
+   ![Ui-doctor](images/Ui-doctor.png)
+   ![Ui-one-patient](images/Ui-one-patient.png)
+   ![Ui-one-doctor](images/Ui-one-doctor.png)
```

Comment:
For the doctor, can we have the start time and end time for each appointment because I think each session doesn't have fixed time frame

Reply:
Okay will do